### PR TITLE
Redesign login page

### PR DESCRIPTION
**Pull request by trenton-wong**

### CODE REVIEW

**Review by achakka007 — COMMENTED** [review]
Good use of flexbox here. Check to see if it works with images as well

**Review by sts04038 — COMMENTED** [review]
great work!

**Review by irinawilliamson — COMMENTED** [review]
Good use of the container. Nice flexibility.

File: code-review/pages/home.jsx [hunk 1]
```diff
@@ -1,36 +1,49 @@
...
+  return (
+    <>
+      <div class="container">
+        <div class="row">
+          <div class="col">
+            <div>
+              <h1 style={{ color: "red" }}>GitHub Code Review:</h1>
+              <h1 style={{ color: "red" }}>GT Edition</h1>
+            </div>
+            <img
```
Comment (awesomelemonade) [inline code-review/pages/home.jsx:25]: per linting, consider using <Image/> from next/image

**Review by awesomelemonade — APPROVED** [review]

**Pull request merged**